chemprop/models/model.py

```diff
@@ -228,41 +228,84 @@ def configure_optimizers(self):
         return {"optimizer": opt, "lr_scheduler": lr_sched_config}
 
     @classmethod
-    def load_submodules(cls, checkpoint_path, **kwargs):
-        hparams = torch.load(checkpoint_path)["hyper_parameters"]
+    def load_submodules(cls, path, map_location, **kwargs):
+        d = torch.load(path, map_location)
+
+        try:
+            hparams = d["hyper_parameters"]
+            state_dict = d["state_dict"]
+        except KeyError:
+            raise KeyError(f"Could not find hyper parameters and/or state dict in {path}.")
+
+        if "message_passing.V_d_transform.mean" in state_dict:
+            dummy_tensor = (
+                torch.zeros_like(state_dict["message_passing.V_d_transform.mean"])
+                .squeeze(0)
+                .numpy()
```

Comment:
`_ScaleTransformMixin` unsqueezes and converts to a torch tensor. If I don't include `.numpy()` here, I get this warning: `UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  scale = torch.cat([torch.ones(pad), torch.tensor(scale, dtype=torch.float)])`